client/settings/settings-manager/index.js

```diff
@@ -100,7 +101,17 @@ const DepositsDescription = () => {
 					depositDelayDays
 				) }
 			</p>
-			<ExternalLink href="https://woocommerce.com/document/payments/faq/deposit-schedule/#section-2">
+			<ExternalLink
+				href="https://woocommerce.com/document/payments/faq/deposit-schedule/#section-2"
+				onClick={ () =>
+					wcpayTracks.recordEvent(
+						wcpayTracks.events
+							.SETTINGS_DEPOSITS_PENDING_SCHEDULES_DOCS_CLICK,
+						'wcpay_deposits_pending_schedules_docs_click',
```

Comment:
I couldn't get this to fire – does `ExternalLink` support an `onClick` handler?

Reply:
I've re-tested and it's not firing for me now either but am confident it was working. The `SETTINGS_DEPOSITS_MANAGE_IN_STRIPE_CLICK` is also using an `ExternalLink` with an `onClick` event and is firing. 
I'm not sure why one is working and the other isn't.

Reply:
I've removed this event as I am unable to resolve this to give consistent tracking.